src/nav/components.yml

```diff
@@ -80,9 +80,9 @@
       id: https://zendeskgarden.github.io/react-components/modals
     - title: Notification
```

Comment:
Should this title be "Notifications"?

Reply:
+1. I just filed a bug bc it looks like we have a lot of inconsistency in plural vs. singular. I think any section that has multiples should receive plural. 

Maybe everything should, but that's for a different PR.